test/unit/host_test.rb

```diff
@@ -2255,12 +2255,14 @@ def to_managed!
     assert_empty host.fact_values.reload
   end
 
-  test 'reports are deleted when build set to true' do
+  test 'reports and host.last_report are deleted when build set to true' do
     host = FactoryGirl.create(:host, :with_reports)
     assert host.reports.present?
     refute host.build?
+    refute host.last_report.blank?
```

Comment:
This assertion is failing, there's no `last_report` in the host. I think it'd be nicer if you split this and made two different tests, one that tests the reports are deleted and other that tests last_report is cleared.